.vscode/extensions.json

```diff
@@ -14,6 +14,7 @@
         "timonwong.shellcheck", // Shell script linting
         "zxh404.vscode-proto3", // Needed for Protobuf linting
         "redhat.vscode-yaml", // YAML formatting
-        "github.vscode-github-actions", // Github Action formatting and linting
+        "github.vscode-github-actions",
+        "bluebrown.yamlfmt", // Github Action formatting and linting
```

Comment:
Is the comment for this on the wrong line? I think it will conflict with the VSCode settings.

Reply:
That might have been a bug with VSCode's "Add to recommended extensions" command. I should file that. Fixed.